demo/README.md

```diff
@@ -1,134 +1,45 @@
 # ArangoDB Spark Datasource Demo
 
-Set ArangoDB Spark Datasource version environment variable:
+Set environment variables:
 
 ```shell
 export ARANGO_SPARK_VERSION=1.0.0
 ```
 
-Set Scala version:
-
-```shell
-# Scala 2.11 is only supported by Spark 2.4
-export SCALA_VERSION=2.11
-
-# Scala 2.12 is supported by both Spark 2.4 and 3.1
-export SCALA_VERSION=2.12
-```
-
-Start ArangoDB cluster:
+Start ArangoDB cluster with docker:
 
 ```shell
 STARTER_MODE=cluster ./docker/start_db.sh
 ```
 
-Import users sample data:
-
-```shell
-./docker/import.sh
-```
-
-## Spark 2.4
+The deployed cluster will be accessible at [http://172.17.0.1:8529](http://172.17.0.1:8529) with username `root` and
+password `test`.
 
 Start Spark cluster:
 
 ```shell
-./docker/start_spark_2.4.sh 
```

Comment:
It could be good to list the requirements for this demo (e.g. Maven, Docker, etc.) with links to their pages? (So that a user who wants to run this demo can get started from scratch)